### PR TITLE
Third task: Adding Amsterdam

### PR DESCRIPTION
**Pull request by BaseMax**

### CODE REVIEW

**Comment by BaseMax** [comment]
review?

**Review by jbampton — COMMENTED** [review]
Cities should be ordered alphabetically

File: doc/source/index.rst [hunk 1]
```diff
@@ -146,11 +146,13 @@ Welcome to slurp's documentation!
    :caption: Cities
    :maxdepth: 2
 
+   amsterdam.md
    amman.md
    andorra.md
```
Comment (jbampton) [inline doc/source/index.rst:151]:
```suggestion
   amman.md
   amsterdam.md
   andorra.md
```

File: doc/source/index.rst [hunk 2]
```diff
@@ -146,11 +146,13 @@ Welcome to slurp's documentation!
...
    belfast.md
    brasilia.md
    brisbane.md
+   buenos-aires.md
    bucharest.md
    budapest.md
```
Comment (jbampton) [inline doc/source/index.rst:157]:
```suggestion
   bucharest.md
   budapest.md
   buenos-aires.md
```

**Comment by BaseMax** [comment]
Thank you for changing.
Merging?

**Comment by jbampton** [comment]
Merged in #1719 👍

**Pull request closed without merging**